code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -1903,9 +1904,13 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 		detach()
 		return
 
+/obj/structure/centrifuge/AltClick()

```

Comment:
It'd be nice if examine gave this kind of meta-info but right now its 100% IC so I'm not so sure